Mapping/Caser.php

```diff
@@ -27,7 +27,7 @@ class Caser
      */
     public static function camel($string)
     {
-        return Inflector::camelize($string);
+        return InflectorFactory::create()->build()->camelize($string);
```

Comment:
Since this is reused wouldn't it be better to have a static variable in the class with the factory being instantialized once? This method is used hundreds of times per run, so wouldn't that be a bit faster?

Reply:
Yeah, probably...